database/influxdb2.go

```diff
@@ -0,0 +1,96 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gobackup/gobackup/helper"
+	"github.com/gobackup/gobackup/logger"
+)
+
+// MongoDB database
+//
+// type: mongodb
+// host: 127.0.0.1
+// port: 27017
+// database:
+// username:
+// password:
+// authdb:
+// exclude_tables:
+// oplog: false
+// args:
+type InfluxDB2 struct {
+	Base
+	host       string
+	token      string
+	bucket     string
+	bucketId   string
+	org        string
```

Comment:
I wanted to avoid the args concept and internal concatenation of arguments (comparing to `Mongodb` that I used as a template), because it does not work well when arguments contain a white space.